vulcanai2/models/basenetwork.py

```diff
@@ -304,20 +298,13 @@ def fit(self, train_loader, val_loader, epochs,
         :param val_loader: The DataLoader object containing the validation data
         :param epochs: The number of epochs
         :param retain_graph: Specifies whether retain_graph will be true when .backwards is called.
-        :param valid_interv:
+        :param valid_interv: Specifies when validation should occur. Not yet implemented.
         :return: None
         """
 
-        self.train_loader = train_loader
-        self.val_loader = val_loader
-        self.epochs = epochs
-        self.retain_graph = retain_graph
-        if valid_interv:
-            self.valid_interv = valid_interv
-
-        self._init_trainer()
+        self._init_trainer(train_loader)
 
-        self.record = dict(
+        record = dict(
```

Comment:
don't we want to hold onto the record? if we load up a model and continue to train it we should be able to see it's previous training cycles through the record @nirtiac . What are your thoughts on this?